modules/create_timepoint/php/create_timepoint.class.inc

```diff
@@ -17,13 +17,12 @@ use \LORIS\StudyEntities\Candidate\CandID;
 /**
  * The forms for the new profile menu
  *
- * @category Loris
+ * @category LORIS_Module
  * @package  Create_Timepoint
- * @author   Loris Team <loris.mni@bic.mni.mcgill.ca>
+ * @author   Alizée Wickenheiser <alizee.wickenheiser@mcin.ca>
  * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  * @link     https://github.com/aces/Loris-Trunk
  */
-
 class Create_Timepoint extends \NDB_Form
 {
     /**
```

Comment:
```suggestion
    public $skipTemplate  = true;
    /**
```

Reply:
you'll can remove the templates directory from this module. in order to do that without errors, you'll need to add the line above ^